lib/config/default.js

```diff
@@ -1185,6 +1185,22 @@ defaultConfig.definition = () => ({
       max_samples_stored: {
         formatter: int,
         default: 10000
+      },
+      labels: {
```

Comment:
Do we not need an option to supply labels to apply?

Reply:
It's going to read the labels from and already defined configuration `labels`